project1/css/index.css

```diff
@@ -24,19 +24,23 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 .container{
-	width: 900px;
-	margin:0 auto;
+	
+	width: 1000px;
+	margin: 0 auto;
 
 }
 header{
-	width: 100%;
+display:flex;
```

Comment:
This display: flex property is causing the navbar to become scrunched together because it thinks it is supposed to be pushing additional elements onto the row. Try removing it and then take a look at the page.